### PR TITLE
Quickstart doc and sample updates (based on internal early accepters experience)

### PR DESCRIPTION
**Pull request by doxiao**
This PR contains the following changes:
1) Add additional info in the quickstart doc and sample readme to help the remote K8S cluster (such as OKE) use cases (based on the experience of a sale person reported in wls-operator-support channel)
2) Tag the generated image with what is specified by the "image" input property
3) Remove a reference to Derby in sample README and properties template file (related to OWLS-70765).
4) Minor clean up to the comments in the sample inputs file.

### CODE REVIEW

File: kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md [hunk 1]
```diff
@@ -60,12 +60,21 @@ If you copy the sample scripts to a different location, make sure that you copy
...
+* A dynamic cluster named `cluster-1` of size 5.
 * Two Managed Servers, named `managed-server1` and `managed-server2`, listening on port `8001`.
 * No applications deployed.
-* A Derby data source.
 * A T3 channel.
 
+If you run the sample from a machine that is remote to the Kubernetes cluster, and you need to push the new image to a registry that is local to the cluster, you need to do the following:
+* Set the `image` property in the inputs file to the target image name (including the tag if needed).
+* Run the `create-domain.sh` script without the `-e` option.
+* Push the generated image `domain-home-in-image-wdt:lastest` (or `domain-home-in-image:latest`) to the target image name.
```
Comment (mriccell) [inline kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md]: Push to the target image or to the repository?

Comment (doxiao) [reply kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md]: Changed to "to the registry".

File: kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md [hunk 2]
```diff
@@ -60,12 +60,21 @@ If you copy the sample scripts to a different location, make sure that you copy
...
+* A dynamic cluster named `cluster-1` of size 5.
 * Two Managed Servers, named `managed-server1` and `managed-server2`, listening on port `8001`.
 * No applications deployed.
-* A Derby data source.
 * A T3 channel.
 
+If you run the sample from a machine that is remote to the Kubernetes cluster, and you need to push the new image to a registry that is local to the cluster, you need to do the following:
+* Set the `image` property in the inputs file to the target image name (including the tag if needed).
+* Run the `create-domain.sh` script without the `-e` option.
+* Push the generated image `domain-home-in-image-wdt:lastest` (or `domain-home-in-image:latest`) to the registry.
```
Comment (simon-meng-cn) [inline kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md]: If set the image property in the inputs file, the property value should be the generated image name. Maybe we need change the statement as following:
Push the generated image `image` (or `domain-home-in-image-wdt:lastest`, or `domain-home-in-image:latest`) to the registry.

site/quickstart.md also need same change.

File: site/quickstart.md [hunk 3]
```diff
@@ -162,7 +162,9 @@ domain namespace (`sample-domain1-ns`) and the `domainHomeImageBase` (`oracle/we
 
 * Setting `weblogicCredentialsSecretName` to the name of the secret containing the WebLogic credentials.
   By convention, the secret will be named`domainUID-weblogic-credentials` (where `domainUID` is replaced with the
-  actual `domainUID` value).
+  actual `domainUID` value). 
+
+* Leaving the `image` empty unless you need to tag the new image that the script builds to a different name.
```
Comment (rosemarymarano) [inline site/quickstart.md]: Leaving the `image` empty -> Leave the `image` empty

Comment (doxiao) [reply site/quickstart.md]: I was trying to follow the existing pattern here. Shall we also change the two bullets above this one too?

Copying the sample -> Copy the sample
Setting weblogicCredentialsSecretName -> Set weblogicCredentialsSecretName

Comment (rosemarymarano) [reply site/quickstart.md]: Oh sorry, I got that wrong, didn't look at the other bulleted items. You were correct to follow the existing pattern of using the gerund form of the verb.

**Pull request merged**